src/app/components/views/home/home.component.html

```diff
@@ -1 +1,47 @@
-<p>home works!</p>
+<div class="container mt-5 mt-xs-1">
+  <div class="row">
+    <div class="col-12 col-md-6 my-auto">
+      <div class="px-3 py-5 px-xs-1 py-xs-1">
+        <h1 class="fw-bold">Store your important urls in a smart way</h1>
+        <p >
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequatur
+          voluptas totam quibusdam nihil! Odio distinctio cum in magni enim
+          facere quaerat minus itaque! Ullam enim obcaecati similique laboriosam
+          tempore beatae.
+        </p>
+      </div>
+    </div>
+    <div class="col-12 col-md-6 d-none d-md-block text-center" >
+      <img
+      style="width: 100%;"
+        class="float-right"
+        src="../../../../assets/designer.png"
+        alt="selecting"
+      />
+    </div>
+  </div>
+  <div class="row mt-5">
+    <div class="col-12 col-md-6 mt-5">
+      <img
+        src="../../../../assets/url.png"
+        alt="new url"
+        style="width: 80%"
+      />
+    </div>
+    <div
+      class="col-12 col-md-6 mt-md-5 my-2 align-self-center"
+      style="
+        align-content: center !important;
+        justify-content: center !important;
+      "
+    >
+      <h1 class="fw-bold text-center">Light , Fast & Powerful</h1>
+      <p class="pt-3 mx-auto d-block">
```

Comment:
This should be same as above p tag, so it shouldn't need extra classes.

Reply:
done